src/src/jwt-verifier.ts

```diff
@@ -0,0 +1,54 @@
+import { Router } from 'express'
+import * as fs from 'fs'
+import * as jws from 'jws'
+import * as moment from 'moment'
+
+const router = Router()
+export default router
+
+const verify = (req, res, next) => {
+    const detachedSignature = req.headers['x-jws-signature']
+    if (!detachedSignature) {
+        errorHandler(req, res, 401, 'INCORRECT_SIGNATURE', 'Signature is empty.')
+        return
```

Comment:
Try reduce empty returns using if/else and moving code around. They makes understanding function flow more difficult.